mindsdb_native/libs/data_sources/file_ds.py

```diff
@@ -186,16 +188,22 @@ def _getDataIo(self, file):
             data.seek(0)
             first_few_lines = []
             i = 0
+
+            # need to have some templates to deduce a dialect
```

Comment:
What do you mean by "templates" here and bellow ? Could you give some examples.

Reply:
under template I mean `sample`
```
class csv.Sniffer
The Sniffer class is used to deduce the format of a CSV file.

The Sniffer class provides two methods:

sniff(sample, delimiters=None)¶
....
```